pkg/sources/kafka/handler.go

```diff
@@ -46,8 +46,15 @@ func newConsumerHandler(readChanSize int) *consumerHandler {
 
 // Setup is run at the beginning of a new session, before ConsumeClaim
 func (consumer *consumerHandler) Setup(sess sarama.ConsumerGroupSession) error {
+	consumer.logger.Info("setup was invoked for a new session")
+	for topic, partitions := range sess.Claims() {
+		println("topic: ", topic)
+		for _, partition := range partitions {
+			println("partition: ", partition)
+		}
+	}
```

Comment:
nit: is this for debugging purpose? can we instead of using println, use logger? Same for below